kubernetes/resource_kubernetes_pod_test.go

```diff
@@ -1204,63 +1200,51 @@ resource "kubernetes_pod" "test" {
 `, secretName, secretName, configMapName, configMapName, podName, imageName)
 }
 
-func testAccKubernetesPodConfigWithInitContainer(podName string, image string) string {
+func testAccKubernetesPodConfigWithInitContainer(podName, image string) string {
 	return fmt.Sprintf(`resource "kubernetes_pod" "test" {
   metadata {
-    labels = {
-      app = "pod_label"
-    }
-
     name = "%s"
   }
 
   spec {
-	automount_service_account_token = false
-
-	container {
-      name  = "nginx"
-      image = "nginx"
-
-      port {
-        container_port = 80
-      }
-
-      volume_mount {
-        name       = "workdir"
-        mount_path = "/usr/share/nginx/html"
-      }
-    }
+  automount_service_account_token = false
+   container {
+     name  = "busybox"
+     image = "busybox"
+     command = ["sh", "-c", "echo The app is running! && sleep 3600"]
+
+     resources {
+       requests {
+         memory = "64Mi"
+         cpu    = "50m"
+       }
+     }
+   }
 
     init_container {
-      name    = "install"
+      name    = "containername"
       image   = "%s"
-      command = ["wget", "-O", "/work-dir/index.html", "http://kubernetes.io"]
+      command = ["sh", "-c", "until nslookup init-service.default.svc.cluster.local; do echo waiting for init-service; sleep 2; done"]
```

Comment:
I replaced our initContainer tests with the example from the Kubernetes site, since ours didn't seem to work anymore.